core/operation/src/main/java/uk/gov/gchq/gaffer/operation/Operation.java

```diff
@@ -233,6 +215,19 @@ default ValidationResult validate() {
         return result;
     }
 
+    default void validateRequiredFieldPresent(final ValidationResult result, final Field field) {
```

Comment:
Can this be private please.

Reply:
I can't make this private until java 9. Is there a better place you can think of to place the concrete method?

Reply:
Ah sorry I forgot this was an interface.  No worries thats fine as it is.